.github/workflows/test-k8s-service-buffer.yml

```diff
@@ -169,6 +169,12 @@ jobs:
           head -c 50000 /tmp/artifacts/test-k8s-state.diff >> /tmp/artifacts/test-k8s-state.diff.message
           echo -e "\n\n(see artifacts in the Github Action for more information)\n\`\`\`\n</details>" >> /tmp/artifacts/test-k8s-state.diff.message
 
+      - name: Dump logs
+        if: always()
+        run: |
+          /tmp/latest-scripts/minikube/logs.sh /tmp/artifacts && 
+          /tmp/latest-scripts/k8s/logs.sh /tmp/artifacts
+
```

Comment:
Dlho som riesil problem, ze mi lokalne, cez MiniKube vsetko ide, ale v CI to neslo.
Preto som sem pridal dump logov, aby som mohol prist na pricinu.